File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.js
function solution(num_list) {
    let num1 = 0;
    let num2 = 0;

    for (let i = 0; i < num_list.length; i++) {
        if (i % 2 === 0) {
            num1 += num_list[i];
        } else {
            num2 += num_list[i];
        }
    }
    
    return num1 > num2 ? num1 : num2;
}

File: 프로그래머스/0/181887. 홀수 vs 짝수/홀수 vs 짝수.py
def solution(num_list):
    sum1 = 0
    sum2 = 0
    for i,num in enumerate(num_list):
        if i % 2 == 0:
            sum1 += num
        else:
            sum2 += num
    return sum1 if sum1 >= sum2 else sum2